core/lexer.h

```diff
@@ -144,66 +144,109 @@ struct Token {
      */
     std::string stringBlockTermIndent;
 
-    UString data32(void) { return decode_utf8(data); }
+    UString data32(void)
+    {
+        return decode_utf8(data);
+    }
 
     LocationRange location;
 
     Token(Kind kind, const Fodder &fodder, const std::string &data,
           const std::string &string_block_indent, const std::string &string_block_term_indent,
           const LocationRange &location)
-      : kind(kind), fodder(fodder), data(data), stringBlockIndent(string_block_indent),
-        stringBlockTermIndent(string_block_term_indent), location(location)
-    { }
+        : kind(kind),
+          fodder(fodder),
+          data(data),
+          stringBlockIndent(string_block_indent),
+          stringBlockTermIndent(string_block_term_indent),
+          location(location)
+    {
+    }
 
-    Token(Kind kind, const std::string &data="")
-        : kind(kind), data(data) { }
+    Token(Kind kind, const std::string &data = "") : kind(kind), data(data) {}
 
     static const char *toString(Kind v)
     {
         switch (v) {
-            case BRACE_L: return "\"{\"";
-            case BRACE_R: return "\"}\"";
-            case BRACKET_L: return "\"[\"";
-            case BRACKET_R: return "\"]\"";
-            case COMMA: return "\",\"";
-            case DOLLAR: return "\"$\"";
-            case DOT: return "\".\"";
-
-            case PAREN_L: return "\"(\"";
-            case PAREN_R: return "\")\"";
-            case SEMICOLON: return "\";\"";
-
-            case IDENTIFIER: return "IDENTIFIER";
-            case NUMBER: return "NUMBER";
-            case OPERATOR: return "OPERATOR";
-            case STRING_SINGLE: return "STRING_SINGLE";
-            case STRING_DOUBLE: return "STRING_DOUBLE";
-            case VERBATIM_STRING_SINGLE: return "VERBATIM_STRING_SINGLE";
-            case VERBATIM_STRING_DOUBLE: return "VERBATIM_STRING_DOUBLE";
-            case STRING_BLOCK: return "STRING_BLOCK";
-
-            case ASSERT: return "assert";
-            case ELSE: return "else";
-            case ERROR: return "error";
-            case FALSE: return "false";
-            case FOR: return "for";
-            case FUNCTION: return "function";
-            case IF: return "if";
-            case IMPORT: return "import";
-            case IMPORTSTR: return "importstr";
-            case IN: return "in";
-            case LOCAL: return "local";
-            case NULL_LIT: return "null";
-            case SELF: return "self";
-            case SUPER: return "super";
-            case TAILSTRICT: return "tailstrict";
-            case THEN: return "then";
-            case TRUE: return "true";
-
-            case END_OF_FILE: return "end of file";
+            case BRACE_L:
```

Comment:
Any way to encourage it to put the return on the same line?  This is less readable than before.